registry/application/services/slack_chat_operation.py

```diff
@@ -204,6 +205,7 @@ def generate_slack_blocks_for_service_listing_template(self, services):
     def process_approval_comment(self, approval_type, state, comment, params):
         if approval_type == "organization":
             org = OrganizationPublisherRepository().get_org_for_org_id(org_id=params["org_id"])
+            orgservice = OrganizationPublisherService(org_uuid=org.uuid, username=org.name)
```

Comment:
rename it to python naming convention `org_service`. use `OrganizationPublisherService("", "")`